.github/workflows/ci.yml

```
@@ -0,0 +1,46 @@
+name: Chai HTTP
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Test on node ${{ matrix.node_version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version:
+          - 8 # to be removed 2020-01
+          - 10 # to be removed 2021-01
+          - x.x.x # safety net; don't remove
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node_version }}
+      - run: npm run env
+      - run: npm i -g npm@6 && npm ci
```

```suggestion
      - run: npm ci
```

If we're dropping support for < 8, npm >= 6 should be installed by default.

---

Oops, wrong - 5 is default on 8.0.0.